dali/pipeline/operators/op_schema.h

```diff
@@ -403,6 +419,8 @@ class DLL_PUBLIC OpSchema {
   bool is_deprecated_;
   string deprecated_in_favor_of_;
 
+  bool no_prune_{};
```

Comment:
Nitpick:
```suggestion
  bool no_prune_ = false;
```
...is more human-readable.
Also, move it next to `is_deprecated_` (above?) to save memory and initialize the latter.